src/Endpoints.ts

```diff
@@ -3,6 +3,7 @@ const regionPlaceholder = '{region}';
 export enum Region {
     US = 'us',
     EU = 'eu',
+    Australia = 'au',
```

Comment:
Just curious, why not `AU`?

Reply:
I was thinking AU wasn't clear enough. AU could be Austria or something else 🤷

Reply:
I found this in wiki https://en.wikipedia.org/wiki/ISO_3166-1
![image](https://user-images.githubusercontent.com/52677246/92524861-cf43e600-f1f0-11ea-9017-047824ed08b2.png)

Reply:
Should I change it?

Reply:
Hmm Let's vote? XD

Reply:
I did the change